src/Formatter/AbstractFormatter.php

```diff
@@ -48,33 +34,4 @@ public static function accepts()
      * @return string
      */
     abstract protected function body(PayloadInterface $payload);
-
-    /**
-     * Get the response status from the payload.
-     *
-     * @param PayloadInterface $payload
-     *
-     * @return integer
-     */
-    public function status(PayloadInterface $payload)
```

Comment:
@shadowhand I have conceptual question here. Can services Responder to modify the Payload object using `Payload::withMessages`? (by convention)

I think that Payload messages is only needed for them and so they can use them for interesting combinations. Then Formatters can modify the output.